src/ServerPrivate.cc

```diff
@@ -43,7 +43,7 @@ struct LoggingPlugin
   {
     static std::string recordPluginFileName =
       std::string("libignition-gazebo") +
```

Comment:
Does the `lib` part need to be removed too?

Reply:
It doesn't change the test results because this is not used for checking against library names in https://github.com/ignitionrobotics/ign-gazebo/blob/4ebe95cebcd5739dbe7c11e5e06e1cdbcfa5fde1/src/ServerPrivate.cc#L204. I went ahead and removed it anyway though. https://github.com/ignitionrobotics/ign-gazebo/pull/335/commits/5032e60a25e9e875de81fd2e9697679b618190e7

Reply:
Thanks! While we're here, could I ask you to change the playback plugin name in `Server.cc` too? :D https://github.com/ignitionrobotics/ign-gazebo/blob/5032e60a25e9e875de81fd2e9697679b618190e7/src/Server.cc#L87-L88

Reply:
Done https://github.com/ignitionrobotics/ign-gazebo/pull/335/commits/9fea17a96f4eec2dce413cdffdb5eeabc151f991. Also renamed the other libraries in the file.